### PR TITLE
text generation: added langchain pipeline script

### PR DESCRIPTION
**Pull request by mgonchar**
Added LangChain script with Q&A example, which supports the same command line options as other generation scripts

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: examples/text-generation/text-generation-pipeline/run_pipeline_langchain.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import argparse
```
Comment (mgonchar) [inline examples/text-generation/text-generation-pipeline/run_pipeline_langchain.py:1]: I don't see any license in scripts within current folder (```examples/text-generation/text-generation-pipeline```), but I see license present in scripts in parent folder ```examples/text-generation```. Please clarify if some license needed

**Review by MohitIntel — COMMENTED** [review]
I tried this change and it works with the following results:

Inference Duration (for 5 iterations): 344.81307863490656 seconds
Throughput: 475.15599074325235 tokens/second

**Review by libinta — APPROVED** [review]

File: examples/text-generation/text-generation-pipeline/run_pipeline_langchain.py (hunk 1 above)
Comment (regisss) [reply examples/text-generation/text-generation-pipeline/run_pipeline_langchain.py:1]: You can add the apache-2.0 license (see an example [here](https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/version.py)) and copyrights for Hugging Face and Intel.

File: examples/text-generation/text-generation-pipeline/README.md [hunk 2]
```diff
@@ -133,37 +133,17 @@ python ../../gaudi_spawn.py --use_deepspeed --world_size 8 run_pipeline.py \
...
-
-prompt = PromptTemplate(input_variables=["question"], template=template)
-llm_chain = LLMChain(prompt=prompt, llm=llm)
-
-# Use LangChain object
-question = "Which libraries and model providers offer LLMs?"
-response = llm_chain(prompt.format(question=question))
-print(f"Question: {question}")
-print(f"Response: {response['text']}")
+To run Q&A example with LangChain use script `run_pipeline_langchain.py` it supports similar syntax to `run_pipeline.py`. For example you can use following command.
```
Comment (regisss) [inline examples/text-generation/text-generation-pipeline/README.md:136]:
```suggestion
To run a Q&A example with LangChain, use the script `run_pipeline_langchain.py`. It supports a similar syntax to `run_pipeline.py`. For example, you can use following command:
```

**Review by regisss — COMMENTED** [review]
LGTM, I just left a comment to refine a bit the README

File: examples/text-generation/text-generation-pipeline/run_pipeline_langchain.py (hunk 1 above)
Comment (mgonchar) [reply examples/text-generation/text-generation-pipeline/run_pipeline_langchain.py:17]: thanks, added in a new commit

File: examples/text-generation/text-generation-pipeline/README.md (hunk 2 above)
Comment (mgonchar) [reply examples/text-generation/text-generation-pipeline/README.md]: thanks, I did this change in latest commit, please have a look

**Review by regisss — APPROVED** [review]

**Pull request merged**